shared/actions/listActions.js

```diff
@@ -99,17 +99,21 @@ export const getDetailImage = (key) => {
     queries.push(createQueryObject('item', key));
     // const address = `${HTTP}${'SERVER_ADDR'}${ENDPOINT_IMAGE}${queryBuilder(queries)}`;
     // const address = `https://goober.herokuapp.com/api/images/${queryBuilder(queries)}`;
-    getAccessToken().then((accessToken) => {
+    return getAccessToken().then((accessToken) => {
       const address = `${HTTPS}${SERVER_ADDR}${ENDPOINT_IMAGE}/${queryBuilder(queries)}`;
       const headers = getAuthHeaders(accessToken);
       return fetch(address, {
         method: 'GET',
         headers
       })
       .then(response => response.json())
-      .then(json =>
-        dispatch(receiveImages(json))
-      );
+      .then(json => {
+        return new Promise((resolve) => {
```

Comment:
Is there a point of returning new promise here when you can simply return and make it thenable?